swcc/swcc/models/project.py

```diff
@@ -402,7 +404,11 @@ def download(self, folder: Union[str, Path]):
             print_progress_bar(0, len(files))
             for index, (path, url) in enumerate(files.items()):
                 file_item: File = File(url)
-                file_item.download(Path(folder, *path.split('/')[:-1]))
+                path_split = path.split('/')
+                file_item.download(
+                    Path(folder, *path_split[:-1]),
+                    file_name=path_split[-1],
+                )
```

Comment:
To confirm: this is the file upload changes we've discussed?

Reply:
Yes, this change was done with the changes to `swcc/swcc/models/file.py` to ensure we download files with the expected file names from the project file (instead of the name of the internally-stored file). 

This became a problem for projects with a custom file name for constraints or landmarks. After editing the data in Cloud, the file name is saved internally as "constraints.json" or "landmarks.csv". This internal name is fine, but when downloading, we needed to save these files with the expected names. Especially in cases with multiple domains, where more than one constraints file and/or more than one landmarks file can exist. If put in the same location with the default file name, one would overwrite the other.